cmd/forward/main.go

```diff
@@ -171,6 +189,16 @@ func Run(_ *cobra.Command, _ []string) {
 		Msgf("finished forwarding queries")
 }
 
+func checkAndCancelQuery(ctx context.Context, queryState *presto.QueryStateInfo) {
+	if queryCacheEntries, ok := runningQueriesCacheMap[queryState.QueryId]; ok {
+		for _, q := range *queryCacheEntries {
+			if q.NextUri != "" {
+				q.Client.CancelQuery(ctx, q.NextUri)
```

Comment:
Let's add some logging in case q.Client.CancelQuery() returns an error

Reply:
Added error log if there is errro returned. Self test passed. please help review again @ethanyzhang . Thanks.